src/styles/themes/default.js

```diff
@@ -10,16 +10,18 @@ export default {
     border: colors.gray2,
     borderFocus: colors.blue,
     icon: colors.gray3,
+    iconSuccessFill: colors.green1,
```

Comment:
I don't anticipate having multiple greens the same way we have a spectrum of grays. So I wonder if it makes more semantic sense to just keep `colors.green` and then use `colors.greenHover` or something like that?

Reply:
Done!